package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "devtools-core",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
devtools-core itself isn't deployed anywhere, I don't think this is necessary to update this version. But I don't mind :)

Reply:
reverted :)